init.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+echo what is your PROJECT_NAME?
```

Comment:
I think `what is your PROJECT_NAME?` should be in quotes. It's weird to see a string like that without quotes.